### DISCUSSION TITLE
Resize overlay_image

### DISCUSSION THREAD

**Discussion by drorcohengithub** [mmistakes/minimal-mistakes#2634] [Q&A] [answered]
I cannot figure out how to resize my homepage image properly.

When I use 

header:
  overlay_image: /assets/images/sketch.JPG

The wrong part of the image is shown - looks like rectangular cropping applied right at the center of the image.

when I use
header:
  image: /assets/images/sketch.JPG

The image is too big and takes up too much of the screen. 

Is there any way to scale the images? If not, what is the correct way to crop/resize the image?

Thanks for the great work

**Comment by mmistakes** [comment, marked as answer]
See #542
Closing as this is a duplicate issue.
